<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarArticulo</class>
 <widget class="QMainWindow" name="AgregarArticulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Atìculo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/imagenes/logoBancomer.jpg</normaloff>:/imagenes/logoBancomer.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background: rgb(255, 255, 255);
}

QPushButton
{
	border-width: 1.5px;
	border-color: rgb(82, 150, 170);
	border-radius:5px;
	padding: 0 8px;
	background-color:rgb(255, 255, 255);
	font: 20pt &quot;Yu Gothic&quot;;

	color:rgb(82, 150, 170)
}

QPushButton:hover
{
	font: 23pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);
}

QGroupBox
{
	border-width: .8px;
	border-color: rgb(82, 150, 170);
	border-style: inset;
	border-radius:5px;
	padding: 0 8px;
	background-color:rgb(255, 255, 255);
	font: 20pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170)
}

QGroupBox QPushButton
{
	border-width: 0px;
	font: 20pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);
}

QGroupBox QPushButton:hover
{
	font: 20pt &quot;Yu Gothic&quot;;
	color:rgb(0, 0, 127)
}

QGroupBox QLabel
{
	border-width: 0px;
	font: 18pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);
}

QLineEdit
{
	border-width: 2px;
	border-color: rgb(82, 150, 170);
	border-style: inset;
	border-radius:10px;
	padding: 0 8px;
	background-color:rgb(255, 255, 255);
}

QTableWidget
{
border-width: 2px;
	border-color: rgb(82, 150, 170);
	border-style: inset;
	border-radius:10px;
	padding: 0 8px;
	background-color:rgb(255, 255, 255);
	font: 75 10pt &quot;Century Gothic&quot;;
	color:rgb(82, 150, 170)	
}
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="barraBBVA">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>47</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="imagenes.qrc">:/imagenes/barraColores.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="logoBBVA">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="imagenes.qrc">:/imagenes/logoBBVA.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">border-width: 0px;
	font: 15pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);</string>
          </property>
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">border-width: 0px;
	font: 15pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);</string>
          </property>
          <property name="text">
           <string>Precio:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>36</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">border-width: 0px;
	font: 15pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);</string>
          </property>
          <property name="text">
           <string>Marca:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>35</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border-width: 0px;
	font: 15pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);</string>
          </property>
          <property name="text">
           <string>Código de barras:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">border-width: 0px;
	font: 15pt &quot;Yu Gothic&quot;;
	color:rgb(82, 150, 170);</string>
          </property>
          <property name="text">
           <string>Cantidad:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>370</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btnAgregar">
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelar">
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>logoBBVA</zorder>
    <zorder>barraBBVA</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>horizontalLayoutWidget_6</zorder>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>btnAgregar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
